pkg/crc/api/api_http.go

```diff
@@ -101,10 +100,6 @@ func pullSecretHandler(config crcConfig.Storage) func(w http.ResponseWriter, r *
 				http.Error(w, err.Error(), http.StatusBadRequest)
 				return
 			}
-			if err := validation.ImagePullSecret(string(bin)); err != nil {
```

Comment:
I think we should keep the check here as well, this differentiate between an invalid pull secret and failing to save the store in the keyring. The tray also depends on this to validate the pull secret and let the user know if the pull secret it invalid

Reply:
Tested, no issues in the tray, the validation is not depending on the returned status code

Reply:
Thanks for the test, the returned error code seemed fairly arbitrary, so I was indeed hoping this would have no impact on the tray.